targets/targets.json

```diff
@@ -3644,9 +3651,7 @@
         "device_has_remove": [
             "LPTICKER"
         ],
-        "release_versions": [
-            "2"
-        ],
+        "supported_application_modes":["bare-metal"],
```

Comment:
Migration guide was updated to use `supported_application_profiles`.

Reply:
[Done](https://github.com/ARMmbed/mbed-os/compare/e443aa7f2073969ff326094d3688c4c7c483a19f..4e61240838a7c9c2add8024c7bcd35ff1d5826b9).